cs-minibot-platform-src/src/main/resources/public/gui/index.html

```diff
@@ -107,9 +107,13 @@
 		<button id="addBot">add bot</button>
 		<button id = "updateLocs"> Run simulator </button>
 		<br><hr><br>
-		Scenario:
-		<textarea rows="4" cols="50" name="scenario" id="scenario">[{"type": "scenario_object", "position":"[5,5]", "angle":45, "size":1},{"type": "scenario_object", "position":"[3,3]", "angle":45, "size": 1}]</textarea>
+		Scenario name:<input type="text" id="scenarioname"
+							 value="scenario1"><br>
+		Scenario viewer: <textarea rows="4" cols="50" name="scenario"
+					   id="scenario">[{"type":	"scenario_object", "position":"[5,5]", "angle":45, "size":1},{"type": "scenario_object", "position":"[3,3]", "angle":45, "size": 1}]</textarea>
```

Comment:
is it possible to load a bot in the scenario? not clear from this json. if so- add a bot to this scenario so it is easy for people to pickup